Resources/Prototypes/_LostParadise/Entities/Clothing/misc.yml

```
@@ -0,0 +1,224 @@
+- type: entity
+  parent: ClothingUniformBase
+  id: LPPClothingJumpsuitMummy
+  name: Mummy bandages
+  description: "They look like ordinary toilet paper rolls and tea bags. But if you soak the former in the latter, you'll get some great aged bandages!"
+  components:
+  - type: Sprite
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)